submissions/kris-almort/html-css-popup/sass/partials/_header.scss

```diff
@@ -0,0 +1,86 @@
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding: 15px 0;
+  position: relative;
+  margin-bottom: 170px;
+}
+
+.top-menu {
+  display: flex;
+  padding: 0 15px;
+  align-items: center;
+
+  &__link-container {
+    line-height: 30px;
+    padding: 0 15px 0 0;
+    font-size: 13px;
+
+    &:last-child {
+      padding: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    transition: .1s;
+    color: rgba(0, 0, 0, 0.87);
+
+    &_notifications {
+      width: 20px;
+      height: 20px;
+      display: block;
+      opacity: .55;
+      cursor: pointer;
+      transition: .2s;
+      background: url("../img/icons-header.png") -694px -89px rgba(0, 0, 0, .55);
+      background-size: 508px 68px;
+      border-radius: 50%;
+
+      &:hover {
+        background-color: rgba(0, 0, 0, .85);
+      }
+    }
+
+    &_profile {
+      display: block;
+      height: 32px;
+      width: 32px;
+      border-radius: 50%;
+      background: url("../img/avatar.jpg") center center no-repeat;
+      background-size: cover;
+    }
+
+
+    &:hover {
+      text-decoration: underline;
+      opacity: .85;
+    }
+  }
+
+  &__popup-input {
+    display: none;
+
```

Comment:
When your element is completely hidden on it the focus does not work and therefore I can’t open it by pressing the button "space". 
You should use properties `visibility: hidden`.

Reply:
[This](https://www.youtube.com/watch?v=E6kLaaQFctU&t=783s) video also can help. Time 29:30.

Reply:
@madmaxWMFU @Temu4 Thanks for noticing. I didnt think about it when I wrote the code. Fix it in commit cb1e64a

Reply:
👍